command/format.go

```diff
@@ -401,6 +401,17 @@ func (t TableFormatter) OutputSecret(ui cli.Ui, secret *api.Secret) error {
 		for k, v := range secret.Auth.Metadata {
 			out = append(out, fmt.Sprintf("token_meta_%s %s %v", k, hopeDelim, v))
 		}
+
+		if secret.Auth.MFARequirement != nil {
+			out = append(out, fmt.Sprintf("mfa_request_id %s %s", hopeDelim, secret.Auth.MFARequirement.MFARequestID))
+
+			for k, constraintSet := range secret.Auth.MFARequirement.MFAConstraints {
```

Comment:
i think we may want to capture the "and" vs "or" requirements for MFA (right now that's just the "any" requirement, which translates to an "or" of the MFA options).

@hghaf099 do you have thoughts on this? might there be a future where MFA requirements can be AND'd together (right now it sounds like they're just OR'd/any'd)?

Reply:
We used to have both "and" and "or" within a single MFAConstraints. However, we deviated from that to just have "or". In cases where a user needs to enforce multiple MFA methods they should define multiple MFAEnforcements.